dash-pipeline/bmv2/dash_inbound.p4

```diff
@@ -36,13 +35,15 @@ control inbound(inout headers_t hdr,
         ConntrackOut.apply(hdr, meta);
 #endif //PNA_CONNTRACK
 
-        vxlan_encap(hdr,
-                    meta.encap_data.underlay_dmac,
-                    meta.encap_data.underlay_smac,
-                    meta.encap_data.underlay_dip,
-                    meta.encap_data.underlay_sip,
-                    hdr.ethernet.dst_addr,
-                    meta.encap_data.vni);
+        tunnel_encap(hdr,
+                     meta,
+                     meta.encap_data.overlay_dmac,
+                     meta.encap_data.underlay_dmac,
+                     meta.encap_data.underlay_smac,
+                     meta.encap_data.underlay_dip,
+                     meta.encap_data.underlay_sip,
+                     dash_encapsulation_t.VXLAN,
```

Comment:
Is there a good reason for using a hardcoded vxlan value here instead of meta.encap_data.dash_encapsulation?

Reply:
I didn't want to change the behavior with this commit. This action can be extended in a separate PR.